.github/workflows/openmetadata-ingestion-core-version-comment.yml

```diff
@@ -37,4 +37,4 @@ jobs:
           body: |
             **Schema Change Detected. Needs py-ingestion-core version bump**
 
-            Please run `make bump-version-dev` in the ingestion-core folder of your repository and commit the changes to _version.py in this PR. Please ignore if this has been handled already.
+            Please run `make core_bump_version_dev` in the ingestion-core folder of your repository and commit the changes to _version.py in this PR. Please ignore if this has been handled already.
```

Comment:
@pmbrull looks great, since we are removing the Makefile from ingestion-core. Can we change the text above to run the command in root folder?

Reply:
Nice catch, thanks